media-library/overview/comments-on-assets.md

```diff
@@ -0,0 +1,69 @@
+**Limits**
+
+- A file can have at most 50 threads.
+- A thread can have at most 50 comments.
+- A comment can have at most 500 characters.
+  {% endhint %}
+
+- **Resolution:** Users have the ability to resolve comments once they have been addressed or no longer require further action. This helps maintain clarity and organization within comment threads.
+
+- **Sorting and filtering:** Users can sort comments by recently created or recent activity, enabling them to prioritize discussions based on their relevance and urgency. Additionally, comments can be filtered by resolved and unresolved threads for easier navigation and management.
+
+![Sort and filter comments as per your need](<../../.gitbook/assets/all-threads-filter.png>)
+
+- **Version control:** Comments are associated with specific versions of files, ensuring that discussions remain linked to the appropriate iteration. This allows for continuity and context preservation, even as files are updated or revised.
+
+![Comments associated with versions](<../../.gitbook/assets/comment-associated-with-versions.png>)
+
+- **Edit and delete:** Users can edit their comments for clarity or correctness and delete them if necessary, ensuring that discussions remain accurate and relevant.
+
+- **Copy and share Link:** Users have the ability to copy and share links to specific threads. This allows them to easily reference discussions or share them with relevant stakeholders.
+
+- **Emoji reactions:** Users can react to comments using emojis, providing a quick and expressive way to acknowledge or respond to comments without adding additional text.
+
+- **Rich text editor:** ImageKit offers a rich text editor, empowering users to enhance their comments with formatting options such as bold, italic, strikethrough, adding links, quoting, and creating bulleted or numbered lists.
+
+- **Emojis in comments:** Users can include emojis within their comments to add expressiveness and convey emotions.
+
+- **Mention other users:** A user can mention other users within the account in their comments, drawing their attention to specific discussions or seeking their input. This fosters collaboration and ensures that relevant stakeholders are engaged in the conversation.
+
+## Watchers and Notifications
+
+- **Watcher Definition:** Watchers of a thread include the owner of the file, anyone participating in the thread by commenting, and users mentioned within the thread discussion. This ensures that relevant stakeholders are kept informed of ongoing discussions and developments.
+
+- **Email Notifications:** Watchers receive email notifications for the following types of new activity on a thread:
+  - Creation of a new thread with comment.
+  - Replies on a thread, facilitating ongoing conversation.
+  - Deletion of a thread.
+  - Resolution or unresolution of a thread, signaling changes in status or resolution.
+
+  Notifications are only sent to users who have at least "Can view" permission on the file.
```

Comment:
Should we mention that user performing the action will not receive an email regarding it?

Reply:
Added